pkg/geoipupdate/geoip_updater.go

```diff
@@ -0,0 +1,118 @@
+package geoipupdate
+
+import (
+	"bytes"
+	"fmt"
+	"github.com/maxmind/geoipupdate/pkg/geoipupdate/database"
+	"github.com/pkg/errors"
+	"io"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"net/url"
+	"path/filepath"
+)
+
+//Run takes the information from a Config and copies all of the provided EditionIDs from a database.Reader to a database.Writer
+func Run(
```

Comment:
I wonder if this should just be part of the main package. Callers could presumably use UpdateEdition() or whatever easily enough.

Reply:
Good call.  Separated this out and shift location of tests to better align with the component being tested.